pkgs/os-specific/linux/kinect-audio-setup/default.nix

```
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchgit
+, requireFile
+, pkg-config
+, libusb1
+, p7zip
+}:
+
+let
+  # The last known good firmware package to have been tested
+  # by the upstream projet.
+  # The firmware URL is hardcoded in the upstream project's installation script
+  firmwareUrl = "https://download.microsoft.com/download/F/9/9/F99791F2-D5BE-478A-B77A-830AD14950C3/KinectSDK-v1.0-beta2-x86.msi";
+  # The original URL "https://research.microsoft.com/en-us/um/legal/kinectsdk-tou_noncommercial.htm"
+  # redirects to the following url:
+  licenseUrl = "https://www.microsoft.com/en-us/legal/terms-of-use";
+in
+stdenv.mkDerivation rec {
+  pname = "kinect-audio-setup";
+  version = "0.5";
```

```suggestion
  # On update: Make sure that the `firmwareURL` is still in sync with upstream.
  # If the project structure hasn't changed you can find the URL in the
  # `kinect_fetch_fw` file in the project source.
  version = "0.5";
```

In case there is another update in the future.

---

Thanks! Fixed